code/stable/nopcm/SRS/NoPCM_SRS.tex

```diff
@@ -528,15 +530,15 @@ \subsubsection{Instance Models}
 \begin{displaymath}
 {m_{W}} {C_{W}} \frac{\,d{T_{W}}}{\,dt}={q_{C}} {A_{C}}
 \end{displaymath}
-Using \hyperref[DD:htFluxC]{DD: htFluxC} , this can be written as:
+Using \hyperref[DD:htFluxC]{DD: htFluxC} for ${q_{C}}$, this can be written as:
 \begin{displaymath}
 {m_{W}} {C_{W}} \frac{\,d{T_{W}}}{\,dt}={h_{C}} {A_{C}} \left({T_{C}}-{T_{W}}\right)
 \end{displaymath}
 Dividing (3) by ${m_{W}} {C_{W}}$, we obtain:
 \begin{displaymath}
 \frac{\,d{T_{W}}}{\,dt}=\frac{{h_{C}} {A_{C}}}{{m_{W}} {C_{W}}} \left({T_{C}}-{T_{W}}\right)
 \end{displaymath}
-Setting ${τ_{W}}$ = ${m_{W}}$ ${C_{W}}$ / ${h_{C}}$ ${A_{C}}$ , Equation (4) can be written in its final form as:
+Setting ${τ_{W}}$ = ${m_{W}}$ ${C_{W}}$ / ${h_{C}}$ ${A_{C}}$, Equation (4) can be written in its final form as:
```

Comment:
Both the old version and the new have `Equation (4)` hard-coded into the tex.  I thought we generated the equation labels and references in LaTeX?